Bottom: 9551487ddb14396d6f59dc049933616d1d826190
Top:    95b49d7ae73f83658145aaaadcc57128a338aa87
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-10-24 19:00:48 +0100

Refresh of board-outline-polygon-generati

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2e2d00a..51821ac 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1393,6 +1393,7 @@ DrawMask (BoxType * screen)
   polygon.Clipped = board_outline_poly ();
   polygon.NoHoles = NULL;
   polygon.NoHolesValid = 0;
+  polygon.BoundingBox = *screen;
   SET_FLAG (FULLPOLYFLAG, &polygon);
   common_fill_pcb_polygon (out->fgGC, &polygon, screen);
   poly_Free (&polygon.Clipped);
